app/uk/gov/hmrc/exports/services/NotificationService.scala

```diff
@@ -106,7 +106,7 @@ class NotificationService @Inject()(submissionRepository: SubmissionRepository,
         val mrn = (singleResponseXml \ "Declaration" \ "ID").text
         val formatter304 = DateTimeFormatter.ofPattern("yyyyMMddHHmmssX")
         val dateTimeIssued =
-          LocalDateTime.parse((singleResponseXml \ "IssueDateTime" \ "DateTimeString").text, formatter304)
+          LocalDateTime.parse((singleResponseXml \ "IssueDateTime" \ "DateTimeString").text, formatter304).atZone(ZoneOffset.UTC).toInstant
```

Comment:
I wonder if `.atZone(ZoneOffset.UTC)` is necessary. We should have the timezone in the field, so parsing and `toInstant` should be enough.

Reply:
Inclined to agree

Reply:
This has been changed...